Tests/MapboxNavigationTests/NavigationViewControllerTests.swift

```diff
@@ -273,21 +331,14 @@ class NavigationViewControllerTests: XCTestCase {
         let route = Fixture.route(from: "DCA-Arboretum", options: options)
         let navigationViewController = NavigationViewController(for: route, routeIndex: 0, routeOptions: options)
         
-        viewController.present(navigationViewController, animated: false, completion: nil)
+        _ = navigationViewController.view
```

Comment:
This is much better than the previous approach of presenting the whole view controller asynchronously.